ember-welcome-page/package.json

```diff
@@ -10,9 +10,19 @@
   "author": "David Baker <acorncom@gmail.com>",
   "exports": {
     ".": "./dist/index.js",
-    "./*": "./dist/*.js",
+    "./*": {
+      "types": "./dist/*.d.ts",
+      "default": "./dist/*.js"
+    },
     "./addon-main.js": "./addon-main.js"
   },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "dist/*"
+      ]
+    }
+  },
```

Comment:
Copy-pasted from the TypeScript project generated by `embroider-build/addon-blueprint`.